client/src/components/HomePage.js

```diff
@@ -52,10 +54,13 @@ const HomePage = () => {
       starArr.push(<StarIcon key={`${recipe.id}-${i}`} className={recipe.rating === 0 ? 'star-icon faded': 'star-icon'}/>);
     }
 
+    // if recipe belongs to multiple categories, use the icon of the first one other than 'All'
+    const icon = recipe.categories.length > 1 ? recipe.categories[1].icon : 'all_icon';
+
     return (
       <div key={recipe.id} className='recipe-card' onClick={() => navigate(`/${recipe.id}`)}>
         <div className='recipe-icon-container'>
-          <img className='recipe-icon' src={allIcon} alt='Icon for all'/>
+          <img className='recipe-icon' src={require(`../graphics/${icon}.svg`)} alt='Icon for the category of the recipe'/>
```

Comment:
Could this alt text have the same format as above? `Icon for category '${category.name}'`

Reply:
Modified